packages/next/build/webpack/plugins/build-manifest-plugin.ts

```diff
@@ -91,17 +92,7 @@ export default class BuildManifestPlugin {
 
           // getFiles() - helper function to read the files for an entrypoint from stats object
           for (const file of entrypoint.getFiles()) {
-            if (/\.map$/.test(file) || /\.hot-update\.js$/.test(file)) {
-              continue
-            }
-
-            // Only `.js` and `.css` files are added for now. In the future we can also handle other file types.
-            if (!/\.js$/.test(file) && !/\.css$/.test(file)) {
-              continue
-            }
-
-            // The page bundles are manually added to _document.js as they need extra properties
-            if (IS_BUNDLED_PAGE_REGEX.exec(file)) {
+            if (!file.endsWith('.js') && !file.endsWith('.css')) {
```

Comment:
IMO this is easier to understand:
```suggestion
            if (!(file.endsWith('.js') || file.endsWith('.css'))) {
```